generate_configurations.py

```diff
@@ -28,7 +28,7 @@ def is_pypy(self):
 PYTHON_VERSIONS = ['python2.6', 'python2.7', 'python3.2', 'python3.3',
                    'python3.4', 'python3.5', 'pypy', 'pypy3']
 PYTEST_VERSIONS = ['2.7.3', '2.8.0']
-DJANGO_VERSIONS = ['1.4', '1.5', '1.6', '1.7', '1.8', '1.9', 'master']
+DJANGO_VERSIONS = ['1.4', '1.5', '1.6', '1.7', '1.8', '1.9',]
```

Comment:
@pelme 
It might work to move `master` to the beginning of the list to still have it in there.
IIRC the list is used from the end when generating variants.